modules/olm-restricted-networks-configuring-operatorhub.adoc

```diff
@@ -112,8 +122,48 @@ command and can be used to further customize the mirroring configuration.
 . If you used the `--manifests-only` flag in the previous step and want to mirror
 only a subset of the content:
 
-.. Modify the list of images in your `mapping.txt` file to your
-specifications.
+.. Modify the list of images in your `mapping.txt` file to your specifications. If
+you are unsure of the exact names and versions of the subset of images you want
+to mirror:
+
+... Run the `sqlite3` tool against the temporary database that was generated by the
+`oc adm catalog mirror` command to retrieve a list of images matching a general
+search query.
++
+For example, to retrieve a list of images that that are similar to the string
+`clusterlogging.4.3`:
++
+[source,terminal]
+----
+$ echo "select * from related_image \
+    where operatorbundle_name like 'clusterlogging.4.3.%';" \
```

Comment:
Hi @adellape  here please change 'clusterlogging.4.3.%' to 'clusterlogging.4.3%' because the above mention "similar to the string
`clusterlogging.4.3`", not "similar to the string`clusterlogging.4.3.`"